CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Breaking
 
+- Use the block's `title` as the source of the translation `id` instead of using the `id` of the block. See upgrade guide for more information. @sneridagh
```

Comment:
Glad that you found this bug!

Correct me if I'm wrong, but the first `id`is too much.
"Use the block's `title` as the source of the translation instead of using the `id` of the block. See upgrade guide for more information. @sneridagh"

Reply:
Done! Thanks!